.travis.yml

```diff
@@ -16,6 +32,12 @@ notifications:
       - https://webhooks.gitter.im/e/d94969c3b01b22cbdcb7
     on_success: change
     on_failure: change
-    on_start: false
-matrix:
-  fast_finish: true
+    on_start: never
+deploy:
+  provider: bintray
+  file: "build/bintray_deploy.json"
+  user: "ctrlaltca"
```

Comment:
Im not sure if creating a fake user is against bintray's TOS: https://bintray.com/docs/terms_of_service.html